tests/testsuite.src/used_binaries.at

```diff
@@ -1139,3 +1139,34 @@ AT_CHECK([$COMPILE_MODULE -Wno-unfinished --copy "f.copy" -fstatic-call prog2.co
 ])
 
 AT_CLEANUP
+
+AT_SETUP([check include header file with directive])
+#AT_KEYWORDS([imp include])
+
+AT_DATA([file.h], [
+COB_EXT_IMPORT void f (char *, long);
+])
+AT_DATA([prog.cob], [
+       >> IMP INCLUDE "file.h"
+       IDENTIFICATION   DIVISION.
+       PROGRAM-ID.      prog.
+       PROCEDURE        DIVISION.
+           CALL "f" USING "Hello".
+])
+
+AT_CHECK([$COBC -m -I . -fstatic-call prog.cob], [1], [], [ignore])
```

Comment:
That duplicates the check for the signature mismatch, as we have that with `--include` already, we can drop it here (note: I hope to check in a change to the `--include` test which _may_ improve the win32 failure this evening).

Reply:
> As the --include test also has --copy covered - please add the >> IMP INCLUDE to the same test, adding IMP INCLUDE directive as keywords

I'm not sure to understand this part. Can you give more details, please?

Reply:
please add this test into the previous one (a separate AT_DATA with AT_TEST call + the two additional entries in AT_KEYWORDS)